packages/components/src/components/Table/TableContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Card } from '../Card/Card';
+
+export const Container = styled.div`
```

Comment:
I know it's a bit of a pain, but I'd really love if we started making the Suite more accessible and use proper semantic elements. Could we use table, thead, tr, th, td and so on, please? Should be fine, if you set the display (flex, block and so on).

Reply:
Good point